src/tasks.js

```diff
@@ -1,19 +1,23 @@
 const generateId = () => (Math.random() + 1).toString(36).substring(7);
 
-export const Task = (description) => {
-  const completedStatus = false;
-  const id = generateId();
+export const Task = (taskData) => {
+  const { description, completedStatus = false, id = generateId() } = taskData;
```

Comment:
- [ ] Do **not** use `Math.random()` numbers for the `id` of the task. The tasks are supposed to have consecutive numbers assigned in the order that the tasks are created. Remember to use a `counter` or `index` variable equal to the length of the array for this, which is updated on add/remove of elements.

Reply:
I disagree with this. Like I said earlier I have created a new method to satisfy the requirement for the counter/index. I am using this Id for other features so I won't be changing it.